### PR TITLE
EWL-6096: hover purple color

### PR DESCRIPTION
**Pull request by damontgomery**
## Ticket(s)
**Jira Ticket**
- [EWL-6096: in body link color](https://issues.ama-assn.org/browse/EWL-6096)

## Description
Explain the technical implementation of the work done.

## To Test
- run `gulp serve`
- View the news article page, http://localhost:3000/?p=pages-news
- Hover over a text link
- View the resource page, http://localhost:3000/?p=pages-resource
- View the inline resource links
- The the inline resource link always has the new color as a background, hover or not

Previous

previous

New

new

## Visual Regressions

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/feature/EWL-6096-hover-purple-color/html_report/index.html).

## Relevant Screenshots/GIFs
See testing steps

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Review by russom-woldezghi — DISMISSED** [review]
Looks good. Text `background-color` changes to #DAD0E1 on hover.

**Review by JaeBea — DISMISSED** [review]
@damontgomery ,
The hoverPurple change is for inline body text only. This updates all things with hoverPurple including the buttons which causes a different accessibility issue on buttons. I think this needs an inlineHoverPurple variable for use only in inline text.

**Comment by damontgomery** [comment]
@JaeBea, thanks for the feedback. I updated this to hopefully apply only to text links in the content area.

@russom-woldezghi, thanks for the review.

**Comment by JaeBea** [comment]
@damontgomery ,
This corrects the on hover color for the link, however there is one other rule that defines link colors on resource page, `.ama__page--resource__resource-link`. This will need the updated to use the new purple variables so that the default state highlighted resource page links are also the correct purple color.

**Comment by damontgomery** [comment]
@JaeBea, thanks. I've included the resource page style in the code and testing steps. Thanks for catching all of this.

**Review by JaeBea — APPROVED** [review]
@damontgomery ,
This looks good to go now. I will approve and then after travis passes, merge into the p2-integration branch.

**Pull request merged**
